wasm/tools/fixtures/runner.py

```diff
@@ -46,24 +46,26 @@
 logger = logging.getLogger("wasm.tools.fixtures.runner")
 
 
+class FloatingPointNotImplemented(NotImplementedError):
```

Comment:
I chose to do it this way to ensure that no other `NotImplementedError` exceptions were caught and interpreted as something that should be ignored.